README.md

```diff
@@ -2,23 +2,72 @@
 
 # [ghuser.io](https://github.com/ghuser-io/ghuser.io)'s database
 
-It is here to cache GitHub's data. During the prototyping phrase, it's just a set of
-[JSON files](data/) that we update once per day by running [this bot](fetchBot/) on an
-[EC2 instance](https://github.com/ghuser-io/ghuser.io/blob/master/aws/ec2).
+This repository:
+* Serves as database for the [ghuser.io](https://github.com/ghuser-io/ghuser.io) Reframe app.\
+The DB consists of the [JSON files in `data`](data/).
+* Provides scripts to update the database. 
+
+In particular, [FetchBot](fetchBot/) runs daily on an [EC2 instance](https://github.com/ghuser-io/ghuser.io/blob/master/aws/ec2).
 
 ## Table of Contents
 
 <!-- toc -->
 
-- [Updating the database](#updating-the-database)
-- [Serving one more user](#serving-one-more-user)
-- [Deleting a profile](#deleting-a-profile)
+- [Setup](#setup)
+- [Usage](#usage)
+- [Implementation](#implementation)
 
 <!-- tocstop -->
 
-## Updating the database
+## Setup
 
-Several scripts form a pipeline for updating the database. Here is the data flow:
+API keys can be created [here](https://github.com/settings/developers).
+
+```bash
+$ npm install
+$ export GITHUB_CLIENT_ID=0123456789abcdef0123
+$ export GITHUB_CLIENT_SECRET=0123456789abcdef0123456789abcdef01234567
+$ export GITHUB_USERNAME=AurelienLourot
+$ export GITHUB_PASSWORD=********
+$ ./fetchAndCalculateAll.sh
```

Comment:
This is not part of the setup, but the actual command for refreshing all profiles.